guides/README.md

```diff
@@ -128,6 +128,7 @@ This makes `git diff` much easier to read and helps reviewing changes.
 * No trailing whitespace on lines and in files.
 Whitespace after partial files has to be handled in the file using the `include::` directive.
 * Source files use UTF-8 character encoding.
+* Image file names use dashes (`-`) and suffix a build target, e.g. `foreman`.
 
```

Comment:
Nice, we could drop a note that we prefer SVG file format for upstream and PNG for downstream? I am actually not sure, SVG works upstream and it's smaller and don't need to have a PNG copy. On the other hand, you probably need more modern browser to open them... like from 2015+ or something. Not big deal.

Reply:
Good point. I can expand the bullet point with my next PR replacing graphics.